source/templates/classic/partials/stargazers.ejs

```diff
@@ -50,6 +50,12 @@
           <% } %>
         </section>
       </div>
+      <% if (plugins.stargazers.worldmap) { %>
+      <div class="row margin-bottom">
+        <h3 class="margin-lr-auto">New stargazers location</h3>
```

Comment:
May be possible that it's actually all stargazers that are fetched, not only the last 14 days, check to change title